igor-web/src/main/groovy/com/netflix/spinnaker/igor/jenkins/JenkinsBuildMonitor.groovy

```diff
@@ -198,8 +198,7 @@ class JenkinsBuildMonitor implements PollingMonitor {
                                                         Project oldProject = new Project(name: project.name, lastBuild: build)
                                                         if (build.number != lastBuild
                                                                 || (build.number == lastBuild && cachedBuild.lastBuildBuilding != build.building)) {
-                                                            echoService.postEvent(
-                                                                    new BuildEvent(content: new BuildContent(project: oldProject, master: master)))
+                                                            postEvent(echoService, cachedBuild, oldProject, master)
```

Comment:
We're on the same page ... what I'm calling `cachedBuilds` is actually the indexed builds for the master.

I've updated the PR to check that.